### PR TITLE
Fix updater file integrity dialog layout

### LINKED ISSUE #20961: File integrity overlay looks ugly on update screen
The update screen performs a file integrity check. If that fails a message is displayed at the bottom of the screen.
This message contains a `Details` link, which should open the detailed results in an overlay.

This overlay looks not correctly designed. Maybe there are some styles not included in the update CSS:

image

### PR DESCRIPTION
**Pull request by bx80**
### Description:

Fixes #20961 

This PR adds missing CSS classes to the file integrity dialog shown on the updater screen and tweaks the style to better match the rest of the UI. Includes a new UI test to prevent future regression.

Before:
image

After:
image

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/CoreUpdater/Controller.php [hunk 1]
```diff
@@ -95,7 +97,7 @@ public function getUpdaterJs()
 
         $files = array(
             "node_modules/jquery/dist/jquery.min.js",
-            "node_modules/jquery-ui-dist/jquery-ui.min.js",
+            "node_modules/jquery-ui-dist/jquery-ui.js",
```
Comment (sgiehl) [inline plugins/CoreUpdater/Controller.php]: any reason for switching to the unminified version?

**Comment by michalkleiner** [comment]
Nice! One thing to note from the 'after' screenshot — the colour of the close button in modals is usually red.

File: plugins/CoreUpdater/Controller.php (hunk 1 above)
Comment (bx80) [reply plugins/CoreUpdater/Controller.php]: I must have edited `getUpdaterJs()` instead of `getUpdaterCss()` at some point. :facepalm: I've reverted it.

**Comment by bx80** [comment]
I overrode the model close button color to green so it matched the color of the other buttons on the page, maybe I should have left it red?

We seem to using a few different variations for modals and I though the red was a bit jarring. Not sure if we have a defined UI standard for modals.

**Comment by michalkleiner** [comment]
Yeah.. the close icon probably should be neutral colour like grey, because it's not a primary action (shouldn't be green) and it's also not an erro/deny action (shouldn't be red), so neither green nor red might be the right one.

I'm happy for you or @sgiehl to make the call.

**Comment by sgiehl** [comment]
Personally I would simply remove the close button. The modal also closes when clicking `OK`, so that's fine. Guess it would also close when pressing `ESC`. So from a UX perspective the close button is imho not needed at all.

**Comment by bx80** [comment]
Sounds good, I've removed the red [X] close icon entirely and the 'Ok' button is still green. 
image

**Comment by sgiehl** [comment]
@bx80 The padding/margin of the headline could maybe be improved so it is align with the text below.

**Comment by bx80** [comment]
@sgiehl I've fixed the title alignment

**Review by sgiehl — APPROVED** [review]
Looks good now 🎉

**Pull request merged**